File: src/targets/node/unirest/fixtures/headers.js
const unirest = require('unirest');

<<<<<<< HEAD
const req = unirest('GET', 'https://httpbin.org/headers');
=======
const req = unirest('GET', 'http://mockbin.com/har');
>>>>>>> upstream/master

req.headers({
  accept: 'application/json',
  'x-foo': 'Bar',
<<<<<<< HEAD
  'x-bar': 'Foo'
=======
  'quoted-value': '"quoted" \'string\''
>>>>>>> upstream/master
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: src/targets/java/unirest/fixtures/headers.java
<<<<<<< HEAD
HttpResponse<String> response = Unirest.get("https://httpbin.org/headers")
  .header("accept", "application/json")
  .header("x-foo", "Bar")
  .header("x-bar", "Foo")
=======
HttpResponse<String> response = Unirest.get("http://mockbin.com/har")
  .header("accept", "application/json")
  .header("x-foo", "Bar")
  .header("quoted-value", "\"quoted\" 'string'")
>>>>>>> upstream/master
  .asString();